src/main/cpp/blaze_util_linux.cc

```diff
@@ -21,6 +21,9 @@
 #include <sys/statfs.h>
 #include <sys/types.h>
 #include <unistd.h>
+#include <stdio.h>
```

Comment:
Is there a particular reason why you included the C headers instead of C++ wrappers like `stdio.h` instead of `cstdio`? The former always feels so wrong when working with C++.